stable/jenkins/values.yaml

```diff
@@ -361,6 +370,7 @@ NetworkPolicy:
 rbac:
   install: false
   serviceAccountName: default
+  serviceAccountAnnotations: {}
```

Comment:
i suggest we keep this PR around Route. there is another PR in progress that handles RBAC.

Reply:
@maorfr I am removing this but this is definitely needed to add TLS support to routes so please get it available asap, Thanks!